mantidproject/mantid
# ISIS_Powder PEARL create_vanadium giving incorrect data

The PEARL routine for creating a vanadium was found to be giving the wrong values. In addition, the `focus_mode` parameter was found to be having no effect. This was pretty much just a matter of things being wired up incorrectly - `create_vanadium` now gives the same results as PEARL's old scripts.

System tests were implemented for all 4 focus modes, the focus mode `groups` was fixed, and some refactoring was done in the existing system tests.

**To test:**

`isis_powder` requires a relatively large amount of setup - come and see me if you take this on and I'll give you a hand setting up all the paths etc...
- As a mininum, a run with the following parameters should not fail:
```
pearl_focus.create_vanadium(run_in_cycle=99481, focus_mode="all", long_mode=False, 
                            tt_mode="tt88", do_absorb_corrections=False)
```
- Play around with some of the other parameters, especially `focus_mode` and see if anything goes bang that shouldn't. To make sure that `focus_mode` is behaving correctly, have a look [here](http://docs.mantidproject.org/nightly/techniques/ISISPowder-Pearl-v1.html#focus-mode)
- Code review

Fixes #20866 

Doesn't need to be in the release notes, as it should have worked all along
